.hashibot.hcl

```diff
@@ -35,6 +35,30 @@ behavior "deprecated_import_commenter" "hashicorp_terraform" {
   EOF
 }
 
+behavior "deprecated_import_commenter" "sdkv1" {
+  import_regexp = "github.com/hashicorp/terraform-plugin-sdk/helper/(schema|resource|acctest)"
+  marker_label  = "terraform-plugin-sdk-v1"
```

Comment:
Can you please add this new label in `infrastructure/repository/labels-workflow.tf` (using same color as the other label)?